### ISSUE TITLE
Running list of minor typos

### ISSUE THREAD

**Issue by Ocean-Minnow** [glacier-modding/H3-Community-Patches#180]
Sometimes the subtitles contain typos. I decided to start keeping a list of them, to be fixed later. I might misquote some of these.

- "Hey. Give me a double **Hurrican**. Lots of rum, no lime." - Doctor in HAVEN, `00EA2F27E8AD7CF9`. Should be [Hurricane](https://en.wikipedia.org/wiki/Hurricane_(cocktail)).
- "I hope no one notices me. I feel like I **realy** stick out here." - Courier, `006F46B88E9C3D57`

Will add more if/when I find them

**Comment by Ocean-Minnow** [comment]
New additions from Splitter patch:

- "[...] wonderful compilation **chuck full** of footage [...]" - "Good Quack Vol. 3" VHS Tape description. Should be "chock full"
- "47 has been **imbedded** undercover [...]" - ICA delegation starting location description. Should be "embedded"

**Comment by VoodooHillbilly** [comment]
- The cloning tank controls prompt reads **\* Turn Off** - including the asterisk.
- Splitter starting point **ICA delegation** should have a capital D.
- The picture-in-picture just reads **Boil** when the Splitter clones are being boiled alive. However, what it should say is going to be subjective.
- I forgot about translations to correct "Good Qua**c**k Vol. 3” VHS Tape.

**Comment by Ocean-Minnow** [comment]
The cloning tank controls read "Increase Temperature" normally, or at least that's what I remember. Something can rarely cause them to stop functioning correctly and display that text, but I don't know what.

I'll add the other ones to the list, as well as two others that I was meaning to add.

**Comment by VoodooHillbilly** [comment]
Not sure. I made a note of it when I saw your screenshot.

**Comment by Ocean-Minnow** [comment]
Ah, I was pointing out that the prompt had changed and the panel had broken :p

**Comment by VoodooHillbilly** [comment]
Couldn't hurt to track down why the prompt breaks. Other than the time spent on it.

**Comment by Ocean-Minnow** [comment]
True. I did give a breakdown of what I did before it broke, when I shared that screenshot. Could create another issue for it.

**Comment by VoodooHillbilly** [comment]
To me, it seems kind of minor to have a separate issue, but idk.

**Comment by VoodooHillbilly** [comment]
The English description for the "A Matter of Loyalty" challenge is `<li>REDACTED.</li>` instead of `<li>[REDACTED]</li>` like other challenges.

**Comment by VoodooHillbilly** [comment]
- officially fixed: remove the English localization for `CHALLENGEPACK_ARGENTUM_WOLVERINEFINISHER_DESC` since IOI's version now matches ours

based on some Spanish translations provided by IWILLCRAFT:
- `UI_PROP_DEVICE_ICA_MODULAR_REMOTE_MUSICDISTRACTION_NAME` change quotes around `«Mixtape 47»`
- in `UI_CHALLENGES_FLU_PATIENT_47_DESC` change `Túmbate en la mesa de una morgue` to `Túmbate en una mesa de la morgue`
- `UI_CONTRACT_TORENIA_BALOON_BRIEFING_DESC` change `a` to `Explota cinco globos.` (yes, it was literally just the letter a)
- `CHAR_REWARD_LEGACY47_STARTINGOUTFIT_M_ PRO225384_NAME` Codename 47 Suit from `El traje Séptimo cielo` (The Seventh Heaven Suit -- why?) to `Traje de Código 47`
- in `CHALLENGEPACK_ARGENTUM_WOLVERINEFINISHER_DESC` change `durante la misión Intocable de los Cárpatos, Rumanía` to `durante la misión «Intocable» en los Cárpatos, Rumanía`
- in `CHALLENGEPACK_ARGENTUM_TIGERDROWNER_DESC` change `Elimina ahogándolo a cualquier objetivo de Bangkok, Tailandia.` to `Elimina a cualquier objetivo en Bangkok, Tailandia ahogándolo.`
- `CHALLENGEPACK_ARGENTUM_TIGERDROWNER_NAME`, `CHALLENGEPACK_ARGENTUM_FOXEXPLODER_DESC`, and `CHALLENGEPACK_ARGENTUM_WRAPPER_DESC` change `Las costumbres del oficio` to `Tradiciones del oficio` and use «» quotes where needed.
- `CHALLENGEPACK_ARGENTUM_FOXEXPLODER_DESC` change `de Berlin` to `en Berlin`
- in `UI_CONTRACT_OPUNTIA_SHURIKENSCARECROW_NAME` and many others, change `Hawke's Bay` to `Bahía de Hawke`

**Comment by VoodooHillbilly** [comment]
#341 has fixed everything currently listed except for the cloning tank `* Turn Off` text. Since a proper fix would also involve changing the logic for making that prompt appear, it wasn't included in that PR.

If you're keeping this issue open, do you want to mark completed items as a checklist the way I have or use some other indicator the typo has been corrected?
